packages/theme/src/cli/services/dev.ts

```diff
@@ -145,6 +129,18 @@ export async function dev(options: DevOptions) {
         break
     }
   })
+
+  await Promise.all([
```

Comment:
> Since the background promise never resolves, Promise.all() waits forever keeping the function alive

Is there a downside to this? Could a user end up with thousands of open promises that never resolve?